docs/docs/spark-procedures.md

```diff
@@ -402,7 +403,8 @@ Iceberg can compact data files in parallel using Spark with the `rewriteDataFile
 | `rewrite-all` | false | Force rewriting of all provided files overriding other options |
 | `max-file-group-size-bytes` | 107374182400 (100GB) | Largest amount of data that should be rewritten in a single file group. The entire rewrite operation is broken down into pieces based on partitioning and within partitions based on size into file-groups.  This helps with breaking down the rewriting of very large partitions which may not be rewritable otherwise due to the resource constraints of the cluster. |
 | `delete-file-threshold` | 2147483647 | Minimum number of deletes that needs to be associated with a data file for it to be considered for rewriting |
-
+| `output-spec-id` | current partition spec id | Desired partition spec ID to be used for rewriting data files. This allows data files to be rewritten with one of existing partition specs. |
+| `remove-dangling-deletes` | false | Remove dangling position and equality deletes after rewriting. A delete file is considered dangling if it does not apply to any live data files. Enabling this will generate an additional snapshot of the delete type. |
```

Comment:
> Enabling this will generate an additional snapshot of the delete type.

1/  if this commit fails does the whole job fails ? 
2/ can you please elaborate on what does `snapshot of delete type` means from what i am understand the operation of this update should be replace as we use `table.newRewrite()` API

Reply:
Agreed, we don't have a "delete" snapshot type.

Reply:
yeah I forgot that I switch to newRewrite instead and I think i can reword it better without mention snapshot type.

With respect to remove dangling delete commit, if there's any runtime failure it can fail the whole job but it will not undo the previous rewrite/compaction. User can either investigate to understand the failure or turn off this feature flag to bypass. Dangling delete removal can also be executed as a standalone SparkAction if needed